ansible/roles/ansible-tvault-contego-extension/tasks/install_rhel.yml

```diff
@@ -15,36 +14,53 @@
 
 - name: Reload Daemon
   shell: systemctl daemon-reload
-
-- name: Generate modules.dep and map files
-  shell: depmod -a
-  register: depmod_reg
+- name: Run cmd to create nbd 128 devices
+  shell: sudo modprobe nbd nbds_max=128
```

Comment:
Will this command run on rhel8 only?

Reply:
@abhijeetpatra said nbd change is only limited to rhel8. Not valid for rhel7.
We need to handle that.